src/bridge/error/error.cc

```diff
@@ -0,0 +1,60 @@
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "src/bridge/error/error.h"
+
+#include <stdio.h>
+
+#include <openssl/err.h>
+
+#include "src/backing/status/status_code.h"
+#include "src/bridge/error/error_strings.h"
+#include "src/bridge/error/function_code.h"
+
+namespace kmsengine {
+namespace bridge {
```

Comment:
For "common utilities" such as error and status, I think it's fine to define them at a top-level namespace.

I'd imagine that to keep remembering to invoke something like `using ::kmsengine::bridge::error` can be pretty clumsy. As long as your have a way to distinguishing OpenSSL errors vs the canonical errors used here, it's fine. I don't think you need to nest that many namespaces.

Reply:
Would it be reasonable to leave the error utilities within `bridge` (but not at `kmsengine`)? (The only things that should use the `error` utilities are things in `bridge` since it directly refers to OpenSSL.)